frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -273,14 +273,6 @@
   }
 
   &__export-btn {
-    margin-left: $pad-medium;
-
-    img {
-      width: 13px;
-      height: 13px;
-      margin-left: 8px;
-      position: relative;
-      top: -2px;
-    }
+    margin-left: $pad-xsmall;
```

Comment:
12px default gap (to match current gap between count and last updated texts)
+ 4px for pad-xsmall = 16px gap (to match current gap between count and button)